trailblazer/apps/tower/api.py

```diff
@@ -77,15 +80,18 @@ def send_request(self, url: str) -> dict:
     @property
     def meets_requirements(self) -> bool:
         """Return True if required variables are not empty."""
-        if self.tower_api_endpoint is None or self.tower_api_endpoint == "":
-            LOG.info("Error: no endpoint specified for Tower Open API request.")
-            return False
-        if self.tower_access_token is None or self.tower_access_token == "":
-            LOG.info("Error: no access token specified for Tower Open API request.")
-            return False
-        if self.workspace_id is None or self.workspace_id == "":
-            LOG.info("Error: no workspace specified for Tower Open API request.")
-            return False
+        requirement_map: List[Tuple[str, str]] = [
+            (self.tower_api_endpoint, "Error: no endpoint specified for Tower Open API request."),
+            (
+                self.tower_access_token,
+                "Error: no access token specified for Tower Open API request.",
+            ),
+            (self.workspace_id, "Error: no workspace specified for Tower Open API request."),
+        ]
+        for requirement, msg in requirement_map:
+            if not requirement:
+                LOG.info(msg)
```

Comment:
```suggestion
                LOG.info(error_msg)
```
Maybe `LOG.warning` or `LOG.error`